<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RcWidget</class>
 <widget class="QWidget" name="RcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 31, 32);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>68</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="MyButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="MyButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>E</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyScroller_Horiz" name="myScroller_Horiz_widget1" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="MyButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyButton" name="pushButton_8">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyButton" name="pushButton_9">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="MyScroller_vertical" name="myScroller_vertical_widget1" native="true">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>180</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>180</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Joystick" name="Joystick_widget1" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>285</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Joystick" name="Joystick_widget2" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>67</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyScroller_Horiz</class>
   <extends>QWidget</extends>
   <header location="global">myscroller_horiz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyScroller_vertical</class>
   <extends>QWidget</extends>
   <header location="global">myscroller_vertical.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Joystick</class>
   <extends>QWidget</extends>
   <header location="global">joystick.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
